src/site/_includes/pattern-set.njk

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+pageScripts:
+  - '/js/content.js'
+---
+
+{% if page.url %}
+  {% set patternSetId = page.url | getRelativePath('/patterns/') %}
+{% endif %}
+
+{% if patternSetId %}
+  {% set patternSet = patterns.sets[patternSetId] %}
+{% endif %}
+
+{% if patternSet.suite %}
+  {% set patternSuite = patterns.sets[patternSet.suite] %}
+  {% set currentPatternSet = patternSet %}
+{% else %}
+  {% set patternSuite = patterns.sets[patternSetId] %}
+{% endif %}
+
+<section class="w-layout-container">
+  {% from 'macros/breadcrumbs.njk' import breadcrumbs with context %}
+  {% set crumbs = [
+    ['All patterns', '/patterns/'],
+    [patternSuite.title, '/patterns/' + patternSuite.id]
+  ] %}
+  {{ breadcrumbs(crumbs) }}
+</section>
+
+<header class="w-masthead-about">
+  <div class="w-masthead-about__container">
+    <div class="w-masthead-about__left">
+      <h1 class="w-headline--two w-font-weight--medium w-mt--non">
+        {{ patternSuite.title }}
+      </h1>
+      <p>
+        {{ patternSuite.description }}
+      </p>
+    </div>
+    <div class="w-masthead-about__right">
+      {% Img
+          height=220,
+          width=330,
+          src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/dYAbN2LdttJ1qYk1gssh.svg",
+          alt="Collection cover image",
+          loading="lazy",
+          decoding="async",
+          class="w-masthead-path__image"
+        %}
+    </div>
+  </div>
+</header>
+
+<hr>
+
+<section class="w-layout-container code-patterns-nav--top">
+  <nav class="code-patterns-nav__sets">
+    {% for setId, set in patterns.sets %}
+      {% if set.suite == patternSuite.id %}
+        {% if setId == currentPatternSet.id %}
+          <a class="selected" href="/patterns/{{set.id}}">{{set.title}}</a>
+        {% else %}
+          <a href="/patterns/{{set.id}}">{{set.title}}</a>
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+  </nav>
+  <div class="code-patterns-nav__filter">
+  <input type="text" placeholder="filter"/>
+  </div>
+</div>
+
+<section class="w-layout-container code-patterns--main">
+  <div>
+    {# Show patterns from current set (if set) or from the whole suite #}
+
+    {% if currentPatternSet %}
```

Comment:
I'm really confused about where this template is used multiple times—seems like one for a whole "suite" and one for a "set"? I can't see it in the `.md` files.

Reply:
Suite is just a special case of a set (it is top level and gets a fancy card in /patterns/). if the page is rendered for a set and not whole suite, their link in menu gets highlighted.

Reply:
I'm also a bit confused between what's a suite and what's a set. It might be good to add a comment somewhere explaining the difference.

It seems like there used to be a suite in this example but it was deleted in this PR?

Reply:
Yes, before there was no distinction between suits & sets and this Pr introduces that distinction.

Reply:
> Yes, before there was no distinction between suits & sets and this Pr introduces that distinction.

What's an example of a suite versus a set?

Reply:
Suite is a top-level set, e.g. /patterns/PatternSuite/PatternSet/Pattern/index.md . It's special power is that it can contain other sets and that it gets a dedicated card in /patterns/ page.